sources/packages/web/src/components/aest/students/assessment/Request.vue

```diff
@@ -81,8 +81,13 @@ export default {
       type: Number,
       required: true,
     },
+    showWhenEmpty: {
```

Comment:
if the show of the base container dependents on this property, then can't the logic be performed by the parent while calling this child? pls let me know if I am missing anything?

Reply:
This component is built in a way that it just takes the applicationId and it makes the API call inside the component. So parent doesn't have any control over the size of the data table. 

In student side of this story, the requests table needs to be hidden when the table is empty. Hence added the logic to component.